src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -116,11 +116,12 @@ def _adapt_settings(settings: Dict) -> Dict:
 def _configure_constants_constsettings(
         scenario: Dict, settings: Dict, connect_to_profiles_db: bool):
     assert isinstance(scenario, dict)
-    if "collaboration_uuid" in scenario or settings.get("type") in [
-            ConfigurationType.CANARY_NETWORK.value, ConfigurationType.B2B.value]:
+    if settings.get("type") in [ConfigurationType.COLLABORATION.value,
```

Comment:
COLLABORATION should not be included here since it is not running in real time.

Reply:
Thanks for catching this. Will do.